setup.sql

```diff
@@ -0,0 +1,10 @@
+/*Operation to create the database*/
+/*CREATE DATABASE userlogin;*/
+\c userlogin;
+
+CREATE TABLE userdata (
+	username VARCHAR(50) PRIMARY KEY,
+	password VARCHAR,
+	loggedin BOOLEAN
```

Comment:
Why do we track whether the user is logged in? What will this be used for?

Reply:
This requirement was included by Richard in the descriptions.